californium-core/src/main/java/org/eclipse/californium/core/coap/ClientObserveRelation.java

```diff
@@ -136,6 +145,26 @@ public ClientObserveRelation(Request request, Endpoint endpoint, ScheduledThread
 		this.request.setProtectFromOffload();
 	}
 
+	/**
+	 * Checks if the relation is reregistered and cancelled confirmable or
+	 * non-confirmable.
+	 *
+	 * @return true, if the relation is registered and cancelled confirmable
+	 */
```

Comment:
Please add a 

```
@since 3.8
```

Reply:
OK

Reply:
Usually, we use 

```
{@code true}
```

If it's OK for you, please change it.

Reply:
It's changed.